scripts/tunnel-creation.mjs

```diff
@@ -0,0 +1,425 @@
+#!/usr/bin/env node
+
+import { logger } from '@appium/support';
+import net from 'node:net';
+
+import {
+  PacketStreamServer,
+  TunnelManager,
+  createLockdownServiceByUDID,
+  createUsbmux,
+  startCoreDeviceProxy,
+  startTunnelRegistryServer,
+} from 'appium-ios-remotexpc';
+
+const log = logger.getLogger('TunnelCreationTest');
+
+/**
+ * Update tunnel registry with new tunnel information
+ * @param {Array} results - Array of tunnel results
```

Comment:
can we have actual types imported from appium-ios-remotexpc instead of just generic objects?